modules/ztp-manually-install-a-single-managed-cluster.adoc

```diff
@@ -271,7 +271,7 @@ spec:
   automatedCleaningMode: disabled
   online: true
 ----
-<1> The baseboard management console address of the installation ISO on the target bare-metal host.
+<1> The baseboard management console (BMC) address of the installation ISO on the target bare-metal host. ZTP supports iPXE and virtual media booting by using Redfish or IPMI protocols. For more information about BMC addressing, see the _Additional resources_ section.
```

Comment:
I know this was here before, but I don't know why this talks about ISO.
This shouldn't have anything to do with what kind of image we're serving, just how ironic accesses the system.

cc @dtantsur